pkg/controller/consoleservice/consoleservice_controller.go

```diff
@@ -10,11 +10,12 @@ import (
 	"encoding/base64"
 	"encoding/json"
 	"fmt"
+	"strconv"
```

Comment:
are not imports supposed to be in alphabetical order ?

Reply:
I let IntelliJ handle that :D